sites/packworld/server/templates/index.marko

```diff
@@ -43,20 +107,98 @@ $ const { id, alias, name, pageNode } = data;
         />
       </div>
       <div class="col-lg-4 mb-block page-rail">
-        <marko-web-gam-display-ad id="gpt-ad-rail2" />
+        <div id="PW-Home-02-Right" />
```

Comment:
These should not have the group prefix, and should be lowercased. `id="home-02-right"`, etc.